apps/labrinth/src/auth/checks.rs

```diff
@@ -364,3 +363,34 @@ pub async fn filter_visible_collections(
 
     Ok(return_collections)
 }
+
+pub async fn is_visible_organization(
+    organization: &DBOrganization,
+    viewing_user: &Option<User>,
+    pool: &PgPool,
+    redis: &RedisPool,
+) -> Result<bool, ApiError> {
+    let members =
+        DBTeamMember::get_from_team_full(organization.team_id, pool, redis)
+            .await?;
+
+    let has_public_or_archived_projects = sqlx::query_scalar!(
+        "SELECT TRUE FROM mods WHERE organization_id = $1 AND status IN ('public', 'archived') LIMIT 1",
+        organization.id as database::models::ids::DBOrganizationId
+    )
+    .fetch_optional(pool)
+    .await?
+    .flatten()
+    .unwrap_or(false);
+
+    let visible = has_public_or_archived_projects
+        || members.iter().filter(|member| member.accepted).count() >= 2
```

Comment:
```suggestion
        || members.iter().filter(|member| member.accepted).count() > 1
```

This makes more sense in my brain 😛